tests/test_parser.py

```diff
@@ -279,3 +280,28 @@ async def test_page_nums():
     assert len(result["segments"]) == 1
     assert "ATTEMPTS" in result["raw"]
     assert "acp@dca.ca.gov" not in result["raw"]
+
+
+@pytest.mark.parametrize(
```

Comment:
We would need to update benchmark table as well right?
Were you planning to do that as a follow-up PR?

Reply:
Ah good point. I'll do it in this PR.